docs/source/guide/tutorials/advanced/self_sl.rst

```diff
@@ -0,0 +1,226 @@
+
+.. note::
+
+  This example demonstates how to work with :doc:`self-supervised learning for classification <../../explanation/algorithms/classification/multi_class_classification>` and :doc:`self-supervised learning for semantic segmentation <../../explanation/algorithms/segmentation/semantic_segmentation>`.
+
+*************************
+Setup virtual environment
+*************************
+
+To create a universal virtual environment for OpenVINO™ Training Extensions, please follow the installation process in the :doc:`quick start guide <../../get_started/quick_start_guide/installation>`.
+
+***************************
+Multi-class classification
+***************************
+.. _tutorial_selfsl_classification:
+
+.. note::
+
+  To prepare `flowers dataset <https://www.tensorflow.org/hub/tutorials/image_feature_vector#the_flowers_dataset>`_, refer to :doc:`how to train classification models <../base/how_to_train/classification>`.
+
+------------
+Pre-training
+------------
+
+1. We can check which models are provided for this task like the command below. All details are in :doc:`how to train classification models <../base/how_to_train/classification>`. We will choose :ref:`MobileNet-V3-large-1x <classificaiton_models>` like :doc:`how to train classification models <../base/how_to_train/classification>`.
+
+.. code-block::
+
+    (otx) ...$ otx find --task classification
+
+    +----------------+---------------------------------------------------+-----------------------+-----------------------------------------------------------------------------------+
+    |      TASK      |                         ID                        |          NAME         |                                        PATH                                       |
+    +----------------+---------------------------------------------------+-----------------------+-----------------------------------------------------------------------------------+
+    | CLASSIFICATION | Custom_Image_Classification_MobileNet-V3-large-1x | MobileNet-V3-large-1x | otx/algorithms/classification/configs/mobilenet_v3_large_1_cls_incr/template.yaml |
+    | CLASSIFICATION |    Custom_Image_Classification_EfficinetNet-B0    |    EfficientNet-B0    |    otx/algorithms/classification/configs/efficientnet_b0_cls_incr/template.yaml   |
+    | CLASSIFICATION |   Custom_Image_Classification_EfficientNet-V2-S   |   EfficientNet-V2-S   |   otx/algorithms/classification/configs/efficientnet_v2_s_cls_incr/template.yaml  |
+    +----------------+---------------------------------------------------+-----------------------+-----------------------------------------------------------------------------------+
+
+2. Prepare an OpenVINO™ Training Extensions workspace by running the following command:
+
+.. note::
+    Unlike :doc:`how to train classification models <../base/how_to_train/classification>`, ``--train-type SELFSUPERVISED`` must be added to get training components for self-supervised learning.
```

Comment:
```suggestion
    Unlike in :doc:`classification tutorial <../base/how_to_train/classification>`, ``--train-type SELFSUPERVISED`` must be added to get training components for self-supervised learning.
```
You don't need to name "how to train classification models" strictly, we can name a link how we want

Reply:
Done